doc/api/next_api_changes/README.rst

```diff
@@ -1,5 +1,9 @@
 :orphan:
 
+.. NOTE
+   Lines 9-end of this file are include in :ref:`api_whats_new`;
+   therefore, please check the doc build after changing this file.
```

Comment:
```suggestion
.. NOTE TO EDITORS OF THIS FILE
   This file serves as README directly available in the file system next to the
   next_api_changes entries. Its content, starting line 11 (after the heading),
   is additionally included in the documentation page doc/devel/api_changes.rst.
   Please check that the page builds correctly after changing this file.
```
I think it helps to be a bit more explicit. Also, the `ref` does not help much
because readers will ever only see this not in source. Instead, state the
file we're embedding in.

When you accept this, please adapt the include line to 11.

Optional alternatives to specifying a start-line, which may get out of sync:
- turn the "heading" into a comment
  ```
  .. Adding API change notes
     =======================
  ```
- Remove the heading from `api_changes.rst` and include the heading from here.